### PR TITLE
SyliusCon advert

### PR DESCRIPTION
**Pull request by Rafikooo**
We'd like to inform as many developers as possible about our second conference fully dedicated to Sylius

image

### CODE REVIEW

**Review by jakubtobiasz — APPROVED** [review]

File: composer.json [hunk 1]
```diff
@@ -24,7 +24,7 @@
         }
     ],
     "require": {
-        "php": "^8.0",
+        "php": "^8.1",
```
Comment (loic425) [inline composer.json]: that's strange to upgrade php version on 1.12 branch?

Comment (jakubtobiasz) [reply composer.json]: https://github.com/Sylius/Sylius-Standard/pull/932#issuecomment-1724887847

Sylius-Standard doesn't follow SemVer (or at least it doesn't have to).

**Review by TheMilek — APPROVED** [review]

File: composer.json (hunk 1 above)
Comment (jakubtobiasz) [reply composer.json]: Anyway, our new teammate @Wojdylak will check what's wrong with 8.0 :p. So we'll wait with merging for results from his investigation.

Comment (loic425) [reply composer.json]: > [#932 (comment)](https://github.com/Sylius/Sylius-Standard/pull/932#issuecomment-1724887847)
> 
> Sylius-Standard doesn't follow SemVer (or at least it doesn't have to).

Yeah indeed, I thought it was on sylius/sylius, misread

Comment (Wojdylak) [reply composer.json]: In php 8.0, the sylius/sylius package installs in version 1.12.9, and there is an issue with doctrine/orm
image

Comment (Rafikooo) [reply composer.json]: Could you add a conflict / write a version of the doctrine package?

Comment (jakubtobiasz) [reply composer.json]: #933 I'm on it

**Comment by jakubtobiasz** [comment]
Thank you, @Rafikooo!

**Pull request merged**